org.eclipse.scout.sdk.s2i/src/main/kotlin/org/eclipse/scout/sdk/s2i/widgetmap/UpdateWidgetMapInScopeMenu.kt

```diff
@@ -17,8 +18,14 @@ import org.eclipse.scout.sdk.s2i.chooseAnalysisScope
 
 class UpdateWidgetMapInScopeMenu : AnAction(message("update.widgetMap.in.scope") + "...", message("update.widgetMap.in.scope.desc"), null) {
 
+    companion object {
+        private val MODULE_SCOPE_TYPES: List<Int> = listOf(AnalysisScope.MODULE, AnalysisScope.MODULES)
```

Comment:
These are the excluded types, right? Maybe reflect that in the name? Otherwise it looks as if these are the used names?